custom/inddex/reports/utils.py

```diff
@@ -15,12 +15,22 @@ class MultiTabularReport(DatespanMixin, CustomProjectReport, GenericTabularRepor
     exportable = True
     exportable_all = True
     export_only = False
+    fix_left_col = True
```

Comment:
This and `fixed_cols_spec` below are to support the `fixColumns`, `fixColsNumLeft`, and `fixColsWidth` values that were previously specified directly in the inddex js.